From b956f2d3a0ef080697253332c990ddfc41bda15e Mon Sep 17 00:00:00 2001
From: Tobias Wiesner <wiesner@lnm.mw.tum.de>
Date: Tue, 25 Nov 2014 23:43:48 +0100
Subject: [PATCH 09/16] Remove dependency to METIS

This cannot go away as long as it is used in the post_drt_common filter code!
---
 src/drt_lib/drt_discret_partition.cpp |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/drt_lib/drt_discret_partition.cpp b/src/drt_lib/drt_discret_partition.cpp
index b4364f7..b203fd6 100644
--- a/src/drt_lib/drt_discret_partition.cpp
+++ b/src/drt_lib/drt_discret_partition.cpp
@@ -17,7 +17,6 @@ Maintainer: Michael Gee
 #include "drt_discret.H"
 #include "drt_exporter.H"
 #include "drt_dserror.H"
-#include "drt_utils_metis.H"
 #include "drt_dofset_pbc.H"
 #include "../linalg/linalg_utils.H"
 
@@ -848,6 +847,9 @@ void DRT::Discretization::SetupGhostingWrongNameDoNotUse(
                                         bool initelements           ,
                                         bool doboundaryconditions   )
 {
+#if 1
+  dserror("SetupGhostingWrongNameDoNotUse not needed here");
+#else
   if (Filled())
     dserror("there is really no need to setup ghosting if the discretization is already filled");
 
@@ -951,6 +953,6 @@ void DRT::Discretization::SetupGhostingWrongNameDoNotUse(
                assigndegreesoffreedom,
                initelements,
                doboundaryconditions);
-
+#endif
 }
 
-- 
1.7.7.6

